docs/src/man/basics.md

```diff
@@ -0,0 +1,2002 @@
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> german.Age # get the vector stored in our DataFrame without copying it
+1000-element Vector{Int64}:
+ 67
+ 22
+ 49
+ 45
+ 53
+ 35
+ 53
+ 35
+ 61
+ 28
+  ⋮
+ 34
+ 23
+ 30
+ 50
+ 31
+ 40
+ 38
+ 23
+ 27
+
+julia> german."Sex" # the same using string
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+
+julia> german[:, 3] # note that this creates a copy
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+```
+
+To get two columns as a `DataFrame`, we can index as follows:
+```jldoctest dataframe
+julia> german[:, 1:2]
+1000×2 DataFrame
+  Row │ id     Age
+      │ Int64  Int64
+──────┼──────────────
+    1 │     0     67
+    2 │     1     22
+    3 │     2     49
+    4 │     3     45
+    5 │     4     53
+    6 │     5     35
+    7 │     6     53
+    8 │     7     35
+  ⋮   │   ⋮      ⋮
+  994 │   993     30
+  995 │   994     50
+  996 │   995     31
+  997 │   996     40
+  998 │   997     38
+  999 │   998     23
+ 1000 │   999     27
+     985 rows omitted
+
+julia> german[:, [3]] # here we have created a single column data frame
+1000×1 DataFrame
+  Row │ Sex
+      │ String
+──────┼────────
+    1 │ male
+    2 │ female
+    3 │ male
+    4 │ male
+    5 │ male
+    6 │ male
+    7 │ male
+    8 │ male
+  ⋮   │   ⋮
+  994 │ male
+  995 │ male
+  996 │ female
+  997 │ male
+  998 │ male
+  999 │ male
+ 1000 │ male
+985 rows omitted
+```
+
+Compare this with accessing the vector contained in column `3`:
+
+```jldoctest dataframe
+julia> german[:, 3]
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+```
+
+We can get a single cell of a data frame with the same syntax to get a cell of a matrix:
+```jldoctest dataframe
+julia> german[4, 4]
+2
+```
+
+or to get a new `DataFrame` that is subset of rows and columns do:
+
+```jldoctest dataframe
+julia> german[4:5, 4:5]
+2×2 DataFrame
+ Row │ Job    Housing
+     │ Int64  String
+─────┼────────────────
+   1 │     2  free
+   2 │     2  free
+```
+
```

Comment:
please move a section on creation of views to this place. Make sure that when you move it you check if all that is presented there makes sense please.

Reply:
If we will move that section here then the order can be a mismatch because now we will move our section before `setindex!` operation. So the order will look like this:
- getindex
- views
- setindex!

What we can do either move this section to that place where we have taken an example of view to create a subdataframe or just for that particular example we can provide the reference of `views` section.

Reply:
As commented above my suggestion is exactly what you have written to have the order:
* `getindex`
* `view`
* `setindex!`

This makes most sense, as `view` and `getindex` are very similar, but just differ in how memory is handled.